scss/base/_base.scss

```diff
@@ -1,41 +1,48 @@
 @font-face { 
-    font-family: 'Georgia';
-    src: url(../fonts/Georgia.ttf);
+  font-family: 'Georgia';
+  src: url(../fonts/Georgia.woff);
```

Comment:
- I mean you must generate font for another browser.
- Using this link to generate it: http://www.font2web.com/